dspace-spring-rest/src/main/java/org/dspace/app/rest/model/SubmissionFormFieldRest.java

```diff
@@ -264,4 +266,11 @@ public void setVisibility(SubmissionVisibilityRest visibility) {
         }
     }
 
+    public List<SelectableRelationship> getSelectableRelationships() {
```

Comment:
Can we add Javadocs to define what a "SelectableRelationship" is?

Reply:
This has been clarified at https://github.com/DSpace/DSpace/blob/969158783f7d819c42e099fd64f60aa71f87c9a4/dspace-server-webapp/src/main/java/org/dspace/app/rest/model/submit/SelectableRelationship.java